haiku-games/gloom3d/gloom3d-1.0.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="Port of the Boom3D Quake-like engine to Haiku"
+DESCRIPTION="Gloom3D is the port Boom3D engine for Haiku. \
+It uses DirectWindow for quick draw window."
+HOMEPAGE="https://github.com/HaikuArchives/Gloom3D"
+COPYRIGHT="2000 Sasha Chukov"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="41b18a6d4038d35b16017251b226eb6c3da778d0"
+SOURCE_URI="https://github.com/HaikuArchives/Gloom3D/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ea920ed4431d781937a06bd79685f99a8acf834c634a00bc37b5a6419cf4d161"
+SOURCE_FILENAME="Gloom3D-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Gloom3D-$srcGitRev"
+
+ARCHITECTURES="?x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	gloom3d = $portVersion
+	app:Gloom3D = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+
```

Comment:
Only 1 empty line here

Reply:
Aside from the empty line this can be merged, @korli could you push the removal of the empty line in case @arnavbhatt288 isn't working on this?